docs/qcvv/parallel_xeb.ipynb

```diff
@@ -303,7 +303,23 @@
    "outputs": [],
```

Comment:
<p>This could be rewritten in slightly more concise way (which makes heatmap_data immutable which is a good property):</p><p> </p><p>heatmap_data = {</p><p>	pair: 1.0 - fidelity for (_, _, pair), fidelity in fidelities.layer_fid.items()</p><p>}</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>